src/PowerShellEditorServices.Engine/Hosting/BuildInfo.cs

```diff
@@ -0,0 +1,9 @@
+namespace Microsoft.PowerShell.EditorServices.Engine.Hosting
+{
+    public static class BuildInfo
+    {
+        public const string BuildVersion = "<development-build>";
+        public const string BuildOrigin = "<development>";
+        public static readonly System.DateTime? BuildTime = System.DateTime.Parse("2019-09-29T23:13:58");
```

Comment:
Why nullable? If `Parse` fails it'll throw during type initialization.

Reply:
@rjmholt do you wanna speak to this?

Reply:
I need to go back and fix the way we handle that file in general, but I think it's nullable so it can be built with null.

I'm not very attached to this idea; it was supposed to be a way to stamp a build to be self-documenting, but I don't think it's ever quite worked, possibly because of git checkin issues.

We can ignore it for now and I'll try and look into it in the coming weeks.

Reply:
Yeah looks like we should just delete this